packages/gatsby-cli/src/structured-errors/error-map.js

```diff
@@ -160,6 +162,34 @@ const errorMap = {
       }`,
     level: `ERROR`,
   },
+  // invalid or deprecated APIs
+  "11329": {
```

Comment:
Is there a pattern to these errors? Were the error codes chosen for a reason, or am I correct in presuming it's effectively an incrementing id that just makes things easier to Google?

Also next-level would be to generate docs pages based on these structured errors, and then provide _even richer detail_ e.g.

>
> To learn more about fixing this error, please visit https://gatsby.dev/11329

Reply:
There is no special meaning to codes. Actually if codes have less patterns the better ;) We want them to be easily google-able (for example to find related issues if linked docs are not enough). So best avoid codes like 11111, 12345 that have higher chance of returning unrelated search results.

One thing that is not really specified, but I think could make sense here, is to not increment by 1 when adding new structured error (if new error is related to different thing than last one).
As example you can check error related to loading gatsby-config - those occupy `10123-10125` range right now and there are slots after that to add more related errors in the future and still keep those grouped.

Reply:
We should just print a prefix maybe then? GTSB-12345 is pretty unique!

Reply:
@pieh circling back to this. Few questions:

> One thing that is not really specified, but I think could make sense here, is to not increment by 1 when adding new structured error

What are you proposing here exactly? 

> those occupy 10123-10125 range right now and there are slots after that to add more related errors in the future and still keep those grouped.

I'm happy to add it there (what you may be saying?) but those are config errors? This could be a gatsby-node.js error, but it wouldn't be a config error necessarily.